src/common/helpers/notify.test.js

```diff
@@ -23,6 +26,10 @@ vi.mock('./logging/logger.js', () => ({
   })
 }))
 
+vi.mock('@defra/cdp-auditing', () => ({
+  audit: (...args) => mockAudit(...args)
+}))
```

Comment:
Nitpick: I appreciate you're following convention here, but I think for terseness you can just do this:

```suggestion
vi.mock('@defra/cdp-auditing', () => ({
  audit: mockAudit
}))
```